toolkit/scripts/pkggen.mk

```diff
@@ -123,6 +124,17 @@ $(cached_file): $(graph_file) $(go-graphpkgfetcher) $(chroot_worker) $(pkggen_lo
 		--output=$(cached_file) && \
 	touch $@
 
+$(scrubbed_file): $(cached_file) $(go-graphscrubber)
+	$(go-graphscrubber) \
+		--input=$(graph_file) \
```

Comment:
Shouldn't the input be `$(cached_file)`? It's even mentioned in this target's dependencies.